index.html

```diff
@@ -85,6 +108,18 @@ <h2>
       Cases</a>
       </li>
     </ul>
+    <h3>
+      Multiple User Agents
+    </h3>
+    <p>
+      In the verifiable digital credentials ecosystem, the digital credential provider, 
+      often known as an identity wallet, functions as a user agent. 
+    </p>
+    <p>
+      When integrated with the Digital Credentials API, 
+      there are two user agents involved: the Web Platform and the digital credential provider. 
```

Comment:
Is "Web Platform" here synonymous with "browser"? Browsers are also called "clients" in similar contexts (e.g. WebAuthn), are other runtimes considered part of "Web Platform" that may prevent a simpler term here?